logger/correlation.go

```diff
@@ -0,0 +1,68 @@
+package logger
+
+import (
+	atomic "github.com/ElrondNetwork/elrond-go/core/atomic"
+)
+
+var correlation logCorrelation
+
+type logCorrelation struct {
+	enabled  bool
+	epoch    atomic.Uint32
+	round    atomic.Int64
+	subRound atomic.String
+}
+
+func (correlation *logCorrelation) enable() {
+	// No need for atomic updates (it should only be called once, in main)
```

Comment:
we should be able to call this as an option from logviewer, also.

Reply:
Fixed :)